src/PaymentGateways/PayPalCommerce/AdminSettingFields.php

```diff
@@ -21,9 +23,64 @@ class AdminSettingFields {
 	 */
 	public function boot() {
 		add_action( 'give_admin_field_paypal_commerce_account_manger', [ $this, 'payPalCommerceAccountManagerField' ] );
+		add_action( 'give_admin_field_paypal_commerce_account_country', [ $this, 'accountCountryField' ] );
 		add_action( 'give_admin_field_paypal_commerce_introduction', [ $this, 'introductionSection' ] );
 	}
 
+	/**
+	 * Render account country field.
+	 *
+	 * @since 2.9.0
+	 */
+	public function accountCountryField() {
+		/* @var MerchantDetail $merchantModel */
+		$merchantModel = give( MerchantDetail::class );
+
+		/* @var MerchantDetails $merchantRepository */
+		$merchantRepository = give( MerchantDetails::class );
+
+		/* @var Give_HTML_Elements $htmlElements */
+		$htmlElements = give( 'html' );
+
+		/* @var Settings $settingRepository */
+		$settingRepository = give( Settings::class );
+
+		$settingHtml = $htmlElements->select(
+			[
+				'id'               => 'paypal_commerce_account_country',
+				'options'          => give_get_country_list(),
+				'chosen'           => true,
+				'placeholder'      => esc_html__( 'Choose a country', 'give' ),
+				'show_option_all'  => false,
+				'show_option_none' => false,
+				'data'             => [
+					'search-type' => 'no_ajax',
+				],
+				'selected'         => $merchantModel->accountCountry ?: $settingRepository->getAccountCountry(),
```

Comment:
@ravinderk Is there any reason to keep the country being stored in `wp_options`? I would think the selected should either be what's in the `MerchantDetail` or the GiveWP country. But setting the country here shouldn't affect the GiveWP country setting. Make sense?

Reply:
@JasonTheAdams I noticed that you add you function to save account country in `wp_options` [`Give\PaymentGateways\PayPalCommerce\Repositories\Settings::updateAccountCountry`](https://github.com/impress-org/givewp/pull/5275/files#diff-2f4056164e4a949b1a5fecaf044f75bfR56) to maintain admin selection even after PayPal account disconnected.

If you want to get ride of this then we can store account country as temporary data and remove it after onboarding admin. Only draw back of this is admin will not able to see last selected country as `selected` in account country setting after disconnecting PayPal account.